arangod/RocksDBEngine/RocksDBEngine.h

```diff
@@ -24,6 +24,7 @@
 
 #ifndef ARANGOD_ROCKSDB_ENGINE_ROCKSDB_ENGINE_H
 #define ARANGOD_ROCKSDB_ENGINE_ROCKSDB_ENGINE_H 1
+#include <rocksdb/write_batch.h>
```

Comment:
Let's not include this, but forward declare it with
```
namespace rocksdb {
class WriteBatch;
}
```

Reply:
If you are concerned about compilation times we should also look into precompiled headers which are available in newer cmake versions.